src/test/resources/testResult/featuresFilter/monitoring-locations/counties_ak_05051.json

```diff
@@ -72,7 +72,7 @@
       "type": "application/geo+json",
       "rel": "self",
       "title": "This document as GeoJSON",
-      "href": "https://unittest/api/observations/collections/monitoring-locations/items?f=json&startIndex=0&limit=10000"
+      "href": "https://unittest/api/observations/collections/monitoring-locations/items?f=json&startIndex=0&limit=10000{FilterOptions}"
```

Comment:
Is the append of filter options here so that unit tests can inject different values?

Reply:
I think I found my answer in a comment above.

Reply:
yes you are correct